### ISSUE TITLE
(URGENT) Citation of "OMG Model Driven Architecture Guide rev. 2.0" crashes

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-omg#5]
Citing this in Metanorma fails:

```adoc
* [[[OMGMDA,OMG Model Driven Architecture Guide rev. 2.0]]] Object Management Group: Model Driven Architecture Guide rev. 2.0
```

Trace:
```
r.rb:295:in `initialize'
	18: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/metanorma-standoc-1.9.0/lib/asciidoctor/standoc/ref_sect.rb:14:in `block in reference'
	17: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/metanorma-standoc-1.9.0/lib/asciidoctor/standoc/ref_sect.rb:14:in `each'
	16: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/metanorma-standoc-1.9.0/lib/asciidoctor/standoc/ref_sect.rb:14:in `block (2 levels) in reference'
	15: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/metanorma-standoc-1.9.0/lib/asciidoctor/standoc/ref.rb:234:in `reference1'
	14: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/metanorma-standoc-1.9.0/lib/asciidoctor/standoc/ref.rb:183:in `refitem'
	13: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/metanorma-standoc-1.9.0/lib/asciidoctor/standoc/ref.rb:191:in `refitem1'
	12: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/metanorma-standoc-1.9.0/lib/asciidoctor/standoc/ref_sect.rb:69:in `fetch_ref'
	11: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/relaton-1.7.8/lib/relaton/db.rb:73:in `fetch'
	10: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/relaton-1.7.8/lib/relaton/db.rb:385:in `check_bibliocache'
	 9: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/relaton-1.7.8/lib/relaton/db.rb:405:in `new_bib_entry'
	 8: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/relaton-1.7.8/lib/relaton/db.rb:420:in `net_retry'
	 7: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/relaton-omg-1.7.2/lib/relaton_omg/processor.rb:18:in `get'
	 6: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/relaton-omg-1.7.2/lib/relaton_omg/omg_bibliography.rb:18:in `get'
	 5: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/relaton-omg-1.7.2/lib/relaton_omg/omg_bibliography.rb:10:in `search'
	 4: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/relaton-omg-1.7.2/lib/relaton_omg/scrapper.rb:14:in `scrape_page'
	 3: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/2.6.0/uri/common.rb:737:in `URI'
	 2: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/2.6.0/uri/common.rb:234:in `parse'
	 1: from /Users/me/.rbenv/versions/2.6.6/lib/ruby/2.6.0/uri/rfc3986_parser.rb:73:in `parse'
/Users/me/.rbenv/versions/2.6.6/lib/ruby/2.6.0/uri/rfc3986_parser.rb:67:in `split': bad URI(is not URI?): "https://www.omg.org/spec/Model/Driven Architecture Guide rev. 2.0" (URI::InvalidURIError)
```

**Comment by andrew2net** [comment]
Fixed in version 1.7.2.
It needs to use acronyms from the list of specs https://www.omg.org/spec/#all
The references should be like OMG + ACRONYM + VERSION. For example `OMG AMI4CCM 1.0`.

**Comment by ronaldtse** [comment]
@andrew2net in any case we cannot crash but have to report to user. Can you confirm that it would not crash in the new implementation? Thanks.

**Comment by ronaldtse** [comment]
Also, could you help document this for users? Thanks!

**Comment by andrew2net** [comment]
> @andrew2net in any case we cannot crash but have to report to user. Can you confirm that it would not crash in the new implementation? Thanks.

It's fixed in #5. I confirm it doesn't crash because of incorrect references.

> Also, could you help document this for users? Thanks!

Done.

**Comment by ronaldtse** [comment]
Thanks!
